generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -102,6 +102,9 @@ import <%= entityAbsolutePackage %>.service.criteria.<%= entityClass %>Criteria;
 <%_ if (searchEngineElasticsearch) { _%>
 import org.assertj.core.util.IterableUtil;
 import org.apache.commons.collections4.IterableUtils;
+import static org.awaitility.Awaitility.await;
```

Comment:
Static import should be moved at the end of the import list

Reply:
done